teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -39,9 +50,13 @@ public int getEth1LogsMaxBlockRange() {
   }
 
   public void configure(final TekuConfiguration.Builder builder) {
+    ArrayList<String> mergedEth1Endpoints = new ArrayList<>();
+    Optional.ofNullable(eth1Endpoint).ifPresent(mergedEth1Endpoints::add);
+    mergedEth1Endpoints.addAll(eth1Endpoints);
+
     builder.powchain(
         b ->
-            b.eth1Endpoint(Optional.ofNullable(eth1Endpoint))
+            b.eth1Endpoints(List.copyOf(mergedEth1Endpoints))
```

Comment:
I wouldn't bother copying the list here.  We're effectively handing over ownership of the list anyway.

Reply:
..there was a reason but.. doesn't apply anymore..